lib/command/src/Obelisk/Command/Utils.hs

```diff
@@ -39,6 +39,9 @@ mvPath = $(staticWhich "mv")
 rmPath :: FilePath
 rmPath = $(staticWhich "rm")
 
+mkdirPath :: FilePath
+mkdirPath = $(staticWhich "mkdir")
```

Comment:
Hm. Why wouldn't we just use haskell to do this ...

Reply:
Well, we'd have to emulate the `mkdir -p` option in this case, as the `mkdir` command line utility is slightly more than a thin wrapper around the `mkdir(2)` syscall.

That's very doable, but would have to be done.  And I didn't really treat this PR as more than a mechanical transformation + should I apply said mechanical transformation in a particular context.

Honestly, I didn't consider _not_ `staticWhich`'ing `mkdir` in this case.  `which` also was a pretty easy call.  I debated briefly regarding `rsync`, `git`, `ssh`, and `ssh-agent`.

Reply:
Oh, I forgot `System.Directory` already has emulation for the `-p` option.

Reply:
Done